keanu-project/src/test/java/io/improbable/keanu/util/csv/CsvWriterTest.java

```diff
@@ -0,0 +1,39 @@
+package io.improbable.keanu.util.csv;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class CsvWriterTest {
+
+    private List<List<String>> data = new ArrayList<>();
+    private String[] row1 = new String[]{"a", "b", "c"};
+    private String[] row2 = new String[]{"a", "b", "c", "d"};
+
+    @Before
+    public void initData() {
+        data.add(Arrays.asList(row1));
+        data.add(Arrays.asList(row2));
+    }
+
+    @Test
+    public void canWriteFile() {
+        CsvWriter writer = new CsvWriter(data);
+        File file = writer.toFile("test", "../");
```

Comment:
test -> test.csv probably. Unless you append .csv on the file name when it's not present in the CsvWriter.

Reply:
nice, gone for test.csv